shell/platform/darwin/macos/framework/Source/FlutterDartProject_Internal.h

```diff
@@ -33,6 +33,13 @@
  */
 @property(nonatomic, readonly) std::vector<const char*> argv;
 
+/**
+ * Instead of looking up the assets and ICU data path in the application bundle, this initializer
+ * allows callers to create a Dart project with custom locations specified for the both.
+ */
+- (_Nonnull instancetype)initWithAssetsPath:(NSString* _Nonnull)assets
```

Comment:
Prefer `nonnull` to `_Nonnull` when possible (such as here) for consistency with the other macOS framework code.